manifests/site.pp

```diff
@@ -61,3 +61,6 @@
   include role::okra
 }
 
+node 'jiraveg' {
```

Comment:
Do you mean that jira should exist in existed server? AFAIR you or @kohsuke said that jira needs separate server.